lib/tictactoe_ui.ex

```diff
@@ -1,88 +1,175 @@
 defmodule TicTacToe.UI do
 
-  def start_game() do
-    TicTacToe.make_move()
-    |> decide_turn_type()
-  end
-  
-  def get_move(game_status) do
-    print_turn(:get_player_move, game_status) 
-  end
+  @clear_screen "\e[H\e[2J"
```

Comment:
There is a function in https://hexdocs.pm/elixir/IO.ANSI.html
that is called `clear/0` that does exactly this.